doc/src/partials/dialogs.vue

```diff
@@ -7,15 +7,15 @@
   .section__content
     p The dialog component allows for verification of user actions, simple data input, and alerts to provide extra information to users. To display dialogs, use events.
 
-    div(v-transfer-dom)
-      mdl-dialog(v-ref:info-message title='Hi there')
+    div
```

Comment:
Argh, I guess that directive is no longer Vue 2 compatible. I'll have to find a solution to make this 2 compatible